src/pages/docs/guides/chat/replies.mdx

```diff
@@ -0,0 +1,803 @@
+---
+title: "Implementing Replies and Quotes with Chat"
+meta_description: "Learn how to implement message replies and quotes in your Ably Chat applications using TypeScript. This guide shows how to use metadata to create quote and reply functionality."
+meta_keywords: "ably chat, message replies, message quotes, chat replies, chat quotes, javascript chat replies, typescript chat replies, chat metadata, ably chat metadata, realtime chat replies"
+---
+
+This guide will show you how you could implement a simple solution for things like replies and quotes in your Ably Chat application.
```

Comment:
I'm going to make the same comment as I did for file uploads in that I think it's immaterial to an end-user that there isn't a dedicated API for handling this. I think this should just sit in the existing 'feature' docs and it isn't necessary to call it out as a guide.